requirements-min.txt

```diff
@@ -1,8 +1,8 @@
-numpy==1.16
```

Comment:
Hm, this should not be here...

Reply:
@achamma723 could you rebase to clean this? Thanks!